### PR TITLE
Initial commit for the formula for kwm

### PR DESCRIPTION
**Pull request by napcae**

### CODE REVIEW

File: Library/Formula/kwm.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  desc "Tiling window manager with focus follows mouse for OSX."
+  homepage "https://koekeishiya.github.io/kwm/"
+  url "https://github.com/koekeishiya/kwm/archive/v1.0.0-RC2.tar.gz"
+  version "1.0.0RC2"
+  sha256 "61252c4710c189cd7f101a3f2555bfdfb2c58c0e32b7856b51bf4e0ba0579202"
+
+  def install
+    system "make", "install"
+    bin.install "bin/kwm"
+    bin.install "bin/kwmc"
```
Comment (bfontaine) [inline Library/Formula/kwm.rb]: Why doesn’t `make install` cover these?

File: Library/Formula/kwm.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
+class Kwm < Formula
+  desc "Tiling window manager with focus follows mouse for OSX."
+  homepage "https://koekeishiya.github.io/kwm/"
+  url "https://github.com/koekeishiya/kwm/archive/v1.0.0-RC2.tar.gz"
```
Comment (bfontaine) [inline Library/Formula/kwm.rb]: We need a stable release here.

File: Library/Formula/kwm.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            <string>IPv4</string>
+        </dict>
+    </dict>
+
+</dict>
+</plist>
+    EOF
+  end
+
+  def caveats; <<-EOF.undent
```
Comment (mikemcquaid) [inline Library/Formula/kwm.rb]: Use `EOS` and put this on the next line to avoid `;`

File: Library/Formula/kwm.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/kwm.rb]: What is `make install` even doing here, given it's not being given a prefix or anything?

File: Library/Formula/kwm.rb [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    system "make", "install"
+    bin.install "bin/kwm"
+    bin.install "bin/kwmc"
+    lib.install "bin/hotkeys.so"
+    prefix.install "examples/kwmrc"
+  end
+
+  def plist; <<-EOF.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```
Comment (DomT4) [inline Library/Formula/kwm.rb]: These few lines need an indent as well, and should be using `EOS.undent`.

File: Library/Formula/kwm.rb [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>com.koekeishiya.kwm</string>
+    <key>ProgramArguments</key>
+    <array>
+      <string>#{bin}/kwm</string>
```
Comment (DomT4) [inline Library/Formula/kwm.rb]: `opt_bin` please.

**Comment by apjanke** [comment]
Our preferred format for the PR title is "kwm 1.0.0-RC2 (new formula)"

**Comment by sdaitzman** [comment]
:+1:

File: Library/Formula/kwm.rb (hunk 2 above)
Comment (antoligy) [inline Library/Formula/kwm.rb]: Here - https://github.com/koekeishiya/kwm/archive/v1.0.4.tar.gz

**Comment by antoligy** [comment]
Versions 1.0.0 through 1.0.4 have since been released - If you are busy @napcae perhaps I can respond to some of the feedback raised in the PR?

**Comment by mikemcquaid** [comment]
@antoligy Sounds good, feel free to open your own PR, thanks!

**Comment by DomT4** [comment]
Closing as dead. Please do feel free to reopen if you wish to with the comments addressed. Thank you for your submission to Homebrew :heart:

**Pull request closed without merging**